data/common.yaml

```diff
@@ -0,0 +1,11 @@
+---
+autofs::package_manage: true
+autofs::package_ensure: 'installed'
+autofs::service_manage: true
+autofs::service_ensure: 'running'
+autofs::service_enable: true
+autofs::service_restart: null
```

Comment:
Does `null` also represent an `undef` in hiera?

I'm just familiar with `~` since it is `nil` in yaml standard like https://github.com/puppetlabs/puppetlabs-ntp/blob/master/data/common.yaml

Reply:
Lessons learned.